{"timestamp":1663270599696,"message":"Deadlock in database layer, retry batch","log.level":"ERROR","logger.name":"managers.db","thread.id":139759070934848,"thread.name":"MainThread","process.id":2590,"process.name":"MainProcess","file.name":"/home/runner/work/drb-etl-pipeline/drb-etl-pipeline/managers/db.py","line.number":54,"entity.type":"SERVICE"}
